### PR TITLE
Task 1 - Initial UI, Name,Health/Mana, Level/Xp

### PR DESCRIPTION
**Pull request by gabrielcicero45**
Adds a `Sheet` component that receives a JSON.
- `Name` deals with the character's name and surname.
- `HealthAndMana`  deals with the health and mana, it calculates the max health/mana based on the character's level.
- `LevelAndXp` deals with the character's level and experience.
Captura de tela de 2023-01-31 09-41-19

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+  left: 8vw;
+}
+
+.img__axe--right{
+  height: 300px;
+  transform: rotateY(180deg);
+  position: absolute;
+  top: 30vh;
+  right: 8vw;
+}
```
Comment (julialbq) [inline src/App.css:23]: Add new line at the end of the file

File: src/components/HealthAndMana.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  const baseMana = 90
+  const maxHealth = baseHealth + (level*10)
+  const maxMana = baseMana + (level*10)
+  return (
+    <>
+    <div className="sheet__item">Health: {health>maxHealth ? maxHealth:health}/{maxHealth}</div>
+    <div className="sheet__item">Mana: {mana>maxMana ? maxMana:mana}/{maxMana}</div>
+    </>
+  )
+}
```
Comment (julialbq) [inline src/components/HealthAndMana.js:14]: Add new line at the end of file

File: src/components/LevelAndXp.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+export default function LevelAndXp({ level, xp }) {
+  return (
+    <>
+      <div className="sheet__item">Level: {level}</div>
+      <div className="sheet__item">Xp: {xp}</div>
+    </>
+  );
+}
```
Comment (julialbq) [inline src/components/LevelAndXp.js:10]: Add new line at the end of file

File: src/components/Name.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+export default function Name({ playerName }) {
+  const name = playerName.split(" ")[0];
+  const surname = playerName.split(" ")[1];
+  return (
+    <>
+      <div className="sheet__item">Name: {name}</div>
+      <div className="sheet__item">Surname: {surname}</div>
+    </>
+  );
+}
```
Comment (julialbq) [inline src/components/Name.js:12]: Add new line at the end of file

File: src/components/Sheet.css [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  margin: auto 24px;
+  background-color: beige;
+  border-radius: 16px;
+}
+
+.sheet__item{
+  text-align: start;
+  padding: 10px;
+  color: #000;
+}
```
Comment (julialbq) [inline src/components/Sheet.css:22]: Add new line at the end of file

File: src/components/Sheet.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+        <HealthAndMana health={player.health} mana={player.mana} level={calculateLevel(player.xp)} />
+      </div>
+      <div className="sheet__column">
+        <LevelAndXp level={calculateLevel(player.xp)} xp={player.xp}/>
+      </div>
+    </div>
+    {player.health<=0 && <h1>You are Dead !</h1>}
+    </>
+  )
+}
```
Comment (julialbq) [inline src/components/Sheet.js:40]: Add new line at the end of file

File: src/components/Sheet.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import HealthAndMana from './HealthAndMana'
+import LevelAndXp from './LevelAndXp'
+import Name from './Name'
+import './Sheet.css'
+export default function Sheet({player}) {
+  const calculateLevel = (xp) => {
```
Comment (julialbq) [inline src/components/Sheet.js:7]: I really liked the idea of the calculateLevel funcition!

File: src/index.css [hunk 8]
```diff
@@ -1,13 +1,7 @@
...
+  font-family: 'Cinzel';
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
-}
+}
```
Comment (julialbq) [inline src/index.css:7]: Add new line at the end of file

**Review by julialbq — APPROVED** [review]

**Review by bioneoficial — APPROVED** [review]

File: src/components/Sheet.css [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+  align-items: center;
+  justify-content: center;
+  margin-top: 30px;
+}
+
+.sheet__column{
+  display: inline-block;
+  font-size: 20px;
+  text-align: center;
+  padding: 10px ;
```
Comment (bioneoficial) [inline src/components/Sheet.css:12]: Please, fix this extra space bettern 'padding: 10px' and the ';'

**Pull request merged**